ui/src/criteria/attribute.tsx

```diff
@@ -46,29 +71,64 @@ class _ implements CriteriaPlugin<Data> {
   }
 
   generateFilter() {
-    if (this.data.selected.length === 0) {
-      return null;
-    }
+    let operands: tanagra.Filter[] | undefined;
+    if (isValid(this.data.min) && isValid(this.data.max)) {
+      operands = [
+        {
+          binaryFilter: {
+            attributeVariable: {
+              variable: "person",
+              name: this.data.attribute,
+            },
+            operator: tanagra.BinaryFilterOperator.LessThan,
+            attributeValue: {
+              int64Val: this.data.max,
+            },
+          },
+        },
+        {
+          binaryFilter: {
+            attributeVariable: {
+              variable: "person",
+              name: this.data.attribute,
+            },
+            operator: tanagra.BinaryFilterOperator.GreaterThan,
```

Comment:
THe PR description says greatThan is not supported yet. Can you update the description?

Reply:
Sure.